pkg/apis/kf/v1alpha1/app_defaults.go

```diff
@@ -32,4 +44,30 @@ func (k *AppSpec) SetDefaults(ctx context.Context) {
 	if len(k.Template.Spec.Containers) == 0 {
 		k.Template.Spec.Containers = append(k.Template.Spec.Containers, corev1.Container{})
 	}
+
+	// Set default disk, RAM, and CPU limits on the application if they have not been custom set
+	k.setResourceLimits(defaultLimits{
+		memory:  defaultMem,
+		storage: defaultStorage,
+		cpu:     defaultCPU,
```

Comment:
I don't think passing this struct is necessary if the fields are private; we can access them directly in `setResourceLimits`

Reply:
done